### PR TITLE
For closing issue #10 && #11, refractor method names.

### PR DESCRIPTION
**Pull request by pushtheworldllc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,12 +65,33 @@ ourBoard.boardConnect(portName).then(function(boardSerial) {
...
+Close the connection with `.streamStop()` and disconnect with `.boardDisconnect()`
+```js
+var ourBoard = new require('openbci-sdk').OpenBCIBoard();
+ourBoard.streamStop().then(ourBoard.disconnect());
+```
+
+Simulating
+----------
+To start the simulator test samples:
+1. Call `.boardSimulateStart()`
```
Comment (teonlamont) [inline README.md]: is this `.boardStimulateStart` or `simulatorStart`?

File: README.md [hunk 2]
```diff
@@ -91,8 +122,8 @@ ourBoard.autoFindOpenBCIBoard(function(portName,ports) {
 Running
 -------
 1. Plug usb module into serial port
-2. Turn OpenBCI device on
-3. Type `npm start` into the terminal in the project directory
+1. Turn OpenBCI device on
+1. Type `npm start` into the terminal in the project directory
```
Comment (teonlamont) [inline README.md]: 1. --> 2.

File: openBCIBoard.js [hunk 3]
```diff
@@ -142,21 +141,80 @@ function OpenBCIFactory() {
...
+        });
+    };
+
+    /**
+     * Purpose: To stop simulating an open bci board
+     * Note: Must be called after the constructor
+     * @returns {Promise}
+     * Author: AJ Keller (@pushtheworldllc)
+     */
+    OpenBCIBoard.prototype.simulateStop = function() {
```
Comment (teonlamont) [inline openBCIBoard.js]: either this should be `simulatorStop` or above should become `simulateStart`

File: package.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "name": "openbci-sdk",
   "version": "0.1.0",
```
Comment (teonlamont) [inline package.json]: does this need to be incremented?

Comment (pushtheworldllc) [inline package.json]: Yes

--------------------------------------------
 
AJ Keller
Founder & CEO
Push The World LLC
Work: (843) 471-0039
Cell: (203) 733-4538

Warning: The information transmitted is intended only for the person or entity to which it is addressed and may contain confidential and/or privileged material. Any review, retransmission, dissemination or other use of, or taking of any action in reliance upon, this information by persons or entities other than the intended recipient is prohibited. If you received this in error, please contact the sender and destroy this material.

File: openBCIBoard.js (hunk 3 above)
Comment (pushtheworldllc) [inline openBCIBoard.js]: It should be simulatorStop

--------------------------------------------
 
AJ Keller
Founder & CEO
Push The World LLC
Work: (843) 471-0039
Cell: (203) 733-4538

Warning: The information transmitted is intended only for the person or entity to which it is addressed and may contain confidential and/or privileged material. Any review, retransmission, dissemination or other use of, or taking of any action in reliance upon, this information by persons or entities other than the intended recipient is prohibited. If you received this in error, please contact the sender and destroy this material.

**Comment by teonlamont** [comment]
hey @pushtheworldllc, once you completed the changes above, I'll merge and version tag repo here

File: README.md (hunk 2 above)
Comment (pushtheworldllc) [inline README.md]: I thought github convention is to put a 1 in front of every list item. When i try to do 2 it does not work

**Comment by teonlamont** [comment]
LGTM, thanks @pushtheworldllc!

**Comment by teonlamont** [comment]
closes #10 and #11

**Pull request merged**